assets/src/components/AssignmentTable.js

```diff
@@ -338,7 +338,7 @@ function AssignmentTable (props) {
         </Grid>
         <Grid item xs={12} sm={3}>
           <FormControl className={classes.formControl}>
-            <InputLabel>Filter by Status</InputLabel>
+            <InputLabel id='assignment-status-checkbox-label'>Filter by Status</InputLabel>
             <Select
```

Comment:
Should both checkboxes label differentiated by type and status? Both seems to have the same Id name.

Reply:
@pushyamig I think the ids are different, `assignment-group-checkbox-label` vs `assignment-status-checkbox-label`. Kind of hard to tell, but they are originally from the id's already written for the <Select> components

Reply:
Thanks for confirming